rule-example.yaml

```diff
@@ -402,3 +402,20 @@
         elements:
           - name: basePackages
             value: "io.konveyor.demo.ordermanagement.repository"
+- ruleID: java-chaining-01
+  category: mandatory
+  description: "There should only be one instance of this rule"
+  message: |
+    Sample message. This rule checks that the chaining conditions are working. Should only get a single issue.
+  when:
+    and:
+      - java.referenced:
+          pattern: java.lang.Override
+          location: ANNOTATION
+        as: class
+        ignore: true
+      - java.referenced:
+          pattern: org.springframework.web.WebApplicationInitializer
+          location: IMPLEMENTS_TYPE
+          filepaths: "{{class.Filepaths}}"
```

Comment:
This is a PITA - Mustache wasn't being able to render `class.filepaths` because `Filepaths` lives inside a struct and is uppercased... We don't really want this, as it is something very implementation related

Reply:
I wonder what happens if you and a Jain strict tag for filepaths that is lowercased in the struct